lib/cosmos/tools/limits_monitor/limits_monitor.rb

```diff
@@ -350,12 +350,13 @@ def reset
       @items = {}
       @out_of_limits = []
       @stale = []
-      @limits_set = get_limits_set()
       unsubscribe_limits_events(@queue_id) if @queue_id
       set_replay_mode(!get_replay_mode()) if @toggle_mode
       @toggle_mode = false
-      @queue_id = subscribe_limits_events(100000)
       @clear_items_callback.call
+
+      @limits_set = get_limits_set()
+      @queue_id = subscribe_limits_events(100000)
```

Comment:
I re-ordered these because it seemed like the get_limits_set was causing a DRbConnError and then it the replay mode never got toggled.